doc/memorydumps.asciidoc

```diff
@@ -29,6 +29,15 @@ without any risk of data changing.
 
 NOTE: Call this method to ensure memory and disk dump refer to the same machine state.
 
+
+=== save_storage
+
+If it is supported by the backend `save_storage` will save all storage disks into backup.
+This can be called anywhere within test. For multiple calls within test module is needed to
+specify different filenames. By default it uses test module name.
+
+NOTE: this method is space and time expensive.
```

Comment:
```suggestion
This can be called from anywhere within a test. For multiple calls within a test module a filename needs to be specified. By default it uses the test module name.

NOTE: This method is time-consuming and requires a lot of space.
```